openlibrary/templates/covers/add.html

```diff
@@ -54,6 +54,24 @@
     </div>
 </form>
 
+<form id="clipboardForm" class="ol-cover-form ol-cover-form--clipboard" method="post" enctype="multipart/form-data" action="$action">
+    <div class="label">
+        <label for="coverClipboard">$_("Or, paste an image from your clipboard")</label>
+    </div>
+
+    <div class="button-container">
+        <img id="image" />
+    </div>
+
+    <input type="file" name="file" id="hiddenFileInput" />
+
+    <div class="button-container">
+        <button id="uploadButtonPaste" type="submit" class="cta-btn cta-btn--vanilla upload-button" aria-label="Use this image"></button>
+        <button type="button" id="pasteButton" class="cta-btn cta-btn--vanilla paste-button" aria-label="Paste image from clipboard"></button>
```

Comment:
```suggestion
        <button type="button" id="pasteButton" class="cta-btn cta-btn--vanilla paste-button" aria-label="$_('Paste image from clipboard')">$_("No image pasted")</button>
        <button id="uploadButtonPaste" type="submit" class="cta-btn cta-btn--vanilla upload-button hidden" aria-label="$_('Upload image')">$_("Upload image")</button>
```
A few changes here:
- Set the button text here instead of in the CSS
- Use i18n strings for the ARIA labels and button text
- Move paste button above upload button
- Use `hidden` class to hide the upload button